### PR TITLE
Fix typo from ElasticSearch to Elasticsearch

### PR DESCRIPTION
**Pull request by kaibadash**
## What type of PR is this? 

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] New Query Runner (Data Source) 
- [ ] New Alert Destination
- [ ] Other

## Description

Fix typo from ElasticSearch to Elasticsearch. 

image

`Elasticsearch` is valid.

## How is this tested?

- [x] Unit tests (pytest, jest)
- [x] E2E Tests (Cypress)
- [x] Manually
    - スクリーンショット 2023-08-06 13 43 00
- [ ] N/A

### CODE REVIEW

**Comment by konnectr** [comment]
What is the meaning of these changes ? For me personally, it makes sense to change the qr name, but not to change the name of the class

**Comment by justinclift** [comment]
It's probably for consistency.  As long as it passes CI and doesn't introduce any other problems, then this change should be ok (in my opinion).  It's making the code base "better". :smile:

**Comment by justinclift** [comment]
@guidopetri @wlach Any interest in taking a look at this PR?

It *seems* pretty straight forward to me. :smile:

**Review by wlach — CHANGES_REQUESTED** [review]
Inclined to agree with @konnectr, changing the display string to match the product name makes sense, but I think using camelcase for the internals makes sense. Certainly doesn't seem worth the risk of breaking things.

**Comment by justinclift** [comment]
Thanks @wlach, sounds like a plan.  :smile:

**Comment by kaibadash** [comment]
Thank you for your review :+1: I'll fix it asap.

File: redash/query_runner/elasticsearch2.py [hunk 1]
```diff
@@ -39,7 +39,7 @@ class ElasticSearch2(BaseHTTPQueryRunner):
 
     @classmethod
     def name(cls):
-        return "ElasticSearch"
+        return "Elasticsearch"
```
Comment (wlach) [inline redash/query_runner/elasticsearch2.py:42]: This is the only change we should be keeping IMO

**Review by wlach — CHANGES_REQUESTED** [review]
Hey, sorry for the lateness in getting back to you on this. Unfortunately, this version has the same problem as before: we're changing the implementation (IMO unnecessary) when all that really needs to be updated is a single display string.

**Comment by kaibadash** [comment]
Thank you for your review!



I think both `OpenDistroSQLElasticSearch` and `XPackSQLElasticSearch` should be corrected. Should I change only `Elasticsearch` ?

**Comment by guidopetri** [comment]
No, I think we can change all 3. Thanks :)

The test error is something unrelated and if you rebase your branch on `master` or merge `master` in, it should be fixed.

**Comment by justinclift** [comment]
This PR confuses me.  Isn't it supposed to only be changing end user visible strings?

However, it looks like it's changing Python class names.

Oh... unless the front end is using the actual Python class name itself as a user visible string?

**Comment by guidopetri** [comment]
Indeed; if you look in line 254 for example, `cls.__name__` returns the python class name. We should change that to hardcode the string in, following @wlach 's advice of not changing the actual class name.

**Review by guidopetri — APPROVED** [review]

**Comment by justinclift** [comment]
@kaibadash Thanks for sticking with this, and sorry we kind of changed things around on you meanwhile. :sweat_smile:

**Pull request merged**